crates/models/src/build/mod.rs

```diff
@@ -225,7 +225,7 @@ pub fn encode_resource_path(resource_path: &[impl AsRef<str>]) -> String {
     for c in parts.join("/").chars() {
         match c {
             // Note that '%' is not included (it must be escaped).
-            '-' | '_' | '+' | '/' | '.' | '=' => name.push(c),
+            '-' | '_' | '+' | '.' | '=' => name.push(c),
```

Comment:
nit: Add a comment here, discussing that '/' is an allowed path symbol but we don't include it because there are further restrictions on uses of '/' which an arbitrary resource path may violate.